src/components/testCaseList/TestCase.tsx

```diff
@@ -56,6 +51,18 @@ const mapGroups = (
   ];
 };
 
+export function getStatusColor(executionStatus: string) {
+  if (executionStatus === "Invalid") {
+    return "secondary";
+  } else if (executionStatus === "NA") {
+    return "default";
+  } else if (executionStatus === "pass") {
+    return "success";
+  } else {
+    return "error";
+  }
+}
```

Comment:
I am wondering if the default statusColor should be either "secondary" or "default" rather than "error".

Reply:
hm we could probably discuss that, I was just maintaining the previous logic. The old logic only had two colors - success and error. If execution status was "pass" then it would use "success" color, otherwise "error" color